docs/src/pages/customization/palette/palette.md

```diff
@@ -68,8 +68,8 @@ const theme = createMuiTheme({
 
 ### Providing the colors directly
 
-If you wish to provide more customized colors, you can either create your own color object,
-or directly supply colors to some or all of the intention's keys:
+If you wish to provide more customized colors, you can either create your own palette color,
```

Comment:
```suggestion
If you wish to provide more customized colors, you can either create your own color object,
```

I may be mistaken, but I think that one was correct.

Reply:
Actually i want to change the next sentence, somehow it got omitted. 
Do you have any suggestions on this one? @mbrookes 
```
or directly supply colors to some or all of the intention's keys:
```

Reply:
or directly supply colors to some or all of the `theme.palette` keys:

?

Reply:
I have no specific preferences. I feel that if we propose custom keys for the second option, we might as well propose the most structured structure for the first one: palette color, which implies the underlying type.

Reply:
Hmm, I find that color object makes sense to me but after reading the page and seeing that we describe the color objects as palette colors we should stay consistent and use "palette colors". 

> palette color, which implies the underlying type.

The underlying type is an object with colours as the values. We even reference this in one of the titles: "Using a color object".